src/pudl/metadata/resources/sec10k.py

```diff
@@ -216,12 +239,14 @@
         "field_namespace": "sec",
     },
     "out_sec10k__changelog_company_name": {
```

Comment:
this is oos for this pr but why the heck are there seemingly idential core and out tables here?